sos.conf

```diff
@@ -1,11 +1,36 @@
-[general]
+[global]
+# Set global options here that are not component specific
+# If you would like one global default value to be specifically overridden for
+# just one component, but not others, you may override that value in the
+# component specific section below
 #verbose = 3
 #verify = yes
 #batch = yes
 #log-size = 15
 
-[plugins]
-#disable = rpm, selinux, dovecot
+[report]
+# Options that will apply to any `sos report` run should be listed here.
+# Note that the option names *must* be the long-form name as seen in --help
```

Comment:
There is something wrong.. 2 issues in fact:

- `sos report has cmd option `--skip-plugins` but we support `noplugins` option in config file.
- `sos report has cmd option `--only-plugins` but we support `onlyplugins` option in config file - without the "`-`". Config file option `only-plugins = kernel` is ignored.

Reply:
Nice catch, we'll need to change the destination in the `add_argument()` statement for these, or change the long form syntax.

Reply:
Rather, the above is just for `-n`. We use `skip-plugins` in the long form, but `noplugins` in code. I think `skip-plugins` would be more favorable here, and I'll go with that if no one objects.